card/lib/card/query.rb

```diff
@@ -97,8 +97,7 @@ def initialize statement
     # Card::Fetch)
 
     def self.run statement
-      query = new statement
-      query.run
+      new(statement.deep_clone).run
```

Comment:
Note that this isn't necessary to fix the specific operator bug, but it's probably wise to prevent other similar problems this way..